kokoro/windows/build.bat

```diff
@@ -51,9 +51,9 @@ REM the more recent ".pkg.tar.zst" packages archives.
 c:\tools\msys64\usr\bin\bash --login -c "pacman -R --noconfirm catgets libcatgets"
 REM Use an old version of patch known to work with the msys runtime
 REM version that comes on Kokoro.
-set REPO_MSYS2=https://sourceforge.net/projects/msys2/files/REPOS/MSYS2/x86_64
+set REPO_MSYS2=http://repo.msys2.org/msys/x86_64/
 set REPO_MINGW=https://sourceforge.net/projects/msys2/files/REPOS/MINGW/x86_64
```

Comment:
We probably want to fetch our packages from the same repo. Now that repo.msys2.org is back (it is the repo that was down for several days), we should probably update both REPO_MSYS2 and REPO_MINGW together, and also use https:

```suggestion
set REPO_MSYS2=https://repo.msys2.org/msys/x86_64/
set REPO_MINGW=https://repo.msys2.org/mingw/x86_64/
```